Content.Shared/Mech/EntitySystems/SharedMechSystem.cs

```diff
@@ -115,6 +115,7 @@ private void RelayInteractionEvent(EntityUid uid, MechComponent component, UserA
     private void OnStartup(EntityUid uid, MechComponent component, ComponentStartup args)
     {
         component.PilotSlot = _container.EnsureContainer<ContainerSlot>(uid, component.PilotSlotId);
+        component.PilotSlot.OccludesLight = false; //starlight
```

Comment:
a bit cursed putting this here but the code sorrounding this is also cursed too so.... sure